sapphire/tests/transformations/test_celestial.py

```diff
@@ -155,11 +171,101 @@ def test_against_jpl_horizons(self):
         ra, dec = celestial.zenithazimuth_to_equatorial(latitude, longitude,
                                                         gps, zenith, azimuth)
 
+        zencalc, azcalc = celestial.equatorial_to_zenithazimuth(latitude,
+                                                                longitude, gps,
+                                                                ra_expected,
+                                                                dec_expected)
+
         self.assertAlmostEqual(ra, ra_expected, 3)
         self.assertAlmostEqual(ra, ra_astropy, 3)
         self.assertAlmostEqual(dec, dec_expected, 2)
         self.assertAlmostEqual(dec, dec_astropy, 2)
 
+        self.assertAlmostEqual(zencalc, zenith, 2)
+        self.assertAlmostEqual(azcalc, azimuth, 2)
+
+class AstropyEquatorialTests(unittest.TestCase):
+    """
+    This tests the 4 new astropy functions. They should be very close to
+    Pyephem results and in this test they are compared to 10 different
+    coordinates from astropy.
+    """
+    def test_pyephem(self):
```

Comment:
Please separate these tests into individual tests, one for each function being tested. That makes it easier to see where a problem exists if the tests fail.